packages/patterns/src/components/create-pattern-modal.js

```diff
@@ -84,7 +99,9 @@ export default function CreatePatternModal( {
 						onChange={ setTitle }
 						placeholder={ __( 'My pattern' ) }
 					/>
-
+					<CategorySelector
+						onCategorySelection={ handleCategorySelection }
+					/>
```

Comment:
If we're updating the create pattern modal in the new patterns package, should we also update the reusable blocks package to maintain parity until it is officially deprecated?

Reply:
I think we should only backport preexisting bug fixes to the `reusable-blocks` package, not new features.

Reply:
So anywhere using the old package can only create uncategorized patterns? Do we an easy way for users to belatedly add categories to a pattern?

Reply:
categories can be modified from `/wp-admin/edit.php?post_type=wp_block`

<img width="794" alt="Screenshot 2023-08-24 at 4 59 18 PM" src="https://github.com/WordPress/gutenberg/assets/3629020/24595175-db22-47bb-9eb4-2d7dcbb00c83">

and we should add that category update option to the inspector panel when editing in the site editor as well.

Reply:
> So anywhere using the old package can only create uncategorized patterns?

Yep, but at this point that is only unknown 3rd party plugins, and we probably shouldn't assume they want the ability to add categories added - if they do they should opt in to using the new patterns package, which we can document in a dev note. I think we should at least add some deprecation warnings to the `reusable-blocks` module as deprecated for 6.4.